earthfile2llb/converter.go

```diff
@@ -147,6 +147,9 @@ func (c *Converter) fromTarget(ctx context.Context, targetName string, platform
 	if err != nil {
 		return errors.Wrapf(err, "apply build %s", depTarget.String())
 	}
+	if mts.Final.RanInteractive {
+		return errors.New("Cannot FROM a target ending with an --interactive")
+	}
```

Comment:
This check was needed because we wouldn't otherwise fail on this case:

```
test:
    FROM alpine:3.13
    RUN apk add bash
    RUN --interactive type=ephemeral bash

from-interactive:
    FROM +test
```